packages/ui/src/components/Select/use-model.ts

```diff
@@ -59,6 +62,10 @@ export function useSelectModel<T = any>({
     return selectedSet.value.has(value)
   }
 
+  function isGroupSelected(optionGroup: SelectOptionGroup<T>) {
+    return optionGroup.items.every((x) => selectedSet.value.has(x.value))
```

Comment:
I think it's better to re-use exising fn:

```suggestion
    return optionGroup.items.every((x) => isValueSelected(x.value))
```